### PR TITLE
Add Task Logic Added (Working)

### PR DESCRIPTION
**Pull request by swlwin**

### CODE REVIEW

**Comment by mattriots** [comment]
This is looking great to me! Thanks for updating the api so it's pointing at the Azure endpoint. Also thanks for changing Stage -> Status and widening the view a bit. Great work. I'm good with merging this request

File: 2do_fe/src/components/AddTask/AddTaskPopUp.js [hunk 1]
```diff
@@ -43,57 +43,78 @@ function AddTaskLogo() {
...
-        // id = "title" />
-      /> */}
           <TextField
-            type="text"
+            required
             className="textfieldstyle"
-            id="title"
+            type="text"
+            name="title"
+            vlaue={taskData.title}
```
Comment (Walter909) [inline 2do_fe/src/components/AddTask/AddTaskPopUp.js:69]: Think this should be value?

**Review by Walter909 — COMMENTED** [review]
I need more clarification on some parts but everything else LGTM

File: 2do_fe/src/components/AddTask/AddTaskPopUp.js (hunk 1 above)
Comment (swlwin) [reply 2do_fe/src/components/AddTask/AddTaskPopUp.js:69]: You're right. That was my bad. Which parts need more clarification?

Comment (swlwin) [reply 2do_fe/src/components/AddTask/AddTaskPopUp.js]: Just fixed the typo!

File: 2do_fe/src/redux/actions/task.actions.js [hunk 2]
```diff
@@ -22,7 +22,29 @@ const getAllTasks = (status, dueDate) => async (dispatch) => {
   }
 };
 
+const addTask = (taskData) => async (dispatch) => {
+  dispatch({type: types.ADD_TASKS_REQUEST, payload: null});
+  try {
+    console.log("Add Tasks", taskData);
```
Comment (katytran) [inline 2do_fe/src/redux/actions/task.actions.js:28]: remove console.log

**Review by katytran — CHANGES_REQUESTED** [review]

File: 2do_fe/src/redux/reducers/index.js [hunk 3]
```diff
@@ -31,7 +31,8 @@ const taskReducer = (state = initialState, action) => {
       };
 
     case types.GET_ALL_TASKS_FAILURE:
-      return { ...state, loading: false };
+    case types.ADD_TASKS_FAILURE:
+      return { ...state, loading: false }; // error: payload
```
Comment (katytran) [inline 2do_fe/src/redux/reducers/index.js:35]: we can remove // error: payload

**Review by katytran — APPROVED** [review]
LGTM

**Review by Walter909 — APPROVED** [review]

**Review by mattriots — APPROVED** [review]

**Pull request merged**